modules/add_metadata_param_tables.R

```diff
@@ -119,19 +119,21 @@ metadata_param_server <- function(id,
         col_group <- data_for_table %>% dplyr::select(group_name, col) %>% unique()
 
         # Select data for in the table to show
-        data_for_table <- metadata_table() %>% dplyr::select(-c(selected, col))
+        data_for_table <- metadata_table() %>%
+          dplyr::select(-c(selected, col, max_obs))
 
-        try(datatable(data_for_table,
+        try(DT::datatable(data_for_table,
                       colnames = c("Station" = "station",
                                    "Group" = "group_name",
-                                   "Percentage measured %" = "per_obs",
-                                   "Maximum measurements" = "max_obs",
+                                   "Datacapture %" = "per_obs",
                                    "First measurements" = "first_m",
                                    "Last measurements" = "last_m",
                                    "Type" = "station_type"),
                       caption = paste0(i18n$t("word_table")," ",
                                        " ", parameter() ," ",i18n$t("word_within"),
-                                       " ",project_or_municipality()),
+                                       " ",project_or_municipality(), "/n for the period: ",
```

Comment:
this linebreak is not working yet.

Reply:
Convert to HTML text? (HTML(".... <br/> for the period: "))

Reply:
https://stackoverflow.com/questions/62654726/print-table-caption-in-two-lines-r-shiny